pkg/analyzer/globalfilechecks/globalfilechecks.go

```diff
@@ -130,6 +135,12 @@ func (state *filePermsType) CheckFile(fi *fsparser.FileInfo, fpath string) error
 		}
 	}
 
+	if state.config.FlagCapabilityInformationalOnly {
+		if len(fi.Capability) > 0 {
```

Comment:
I think we should do `fi.Capability == ""` since it's more clear (to me, at least)

Reply:
this is an array